_std/macros/atmos.dm

```diff
@@ -334,3 +334,24 @@ proc/gas_text_color(gas_id)
 
 #define _LIST_CONCENTRATION_REPORT(GAS, _, NAME, MIXTURE, LIST) LIST += "[NAME]: [round(MIXTURE.GAS / total_moles * 100)]%";
 #define LIST_CONCENTRATION_REPORT(MIXTURE, LIST) APPLY_TO_GASES(_LIST_CONCENTRATION_REPORT, MIXTURE, LIST)
+
+//Possible states are exposed and intact. sizes are short, medium and long.
+#define SET_PIPE_UNDERLAY(NODE, DIR, SIZE, COLOUR, HIDDEN) do { \
+	if (UNLINT(HIDDEN)) { \
```

Comment:
What's this unlint doing?

Reply:
some devices never have their pipe change visibility, so we set them as FALSE. Unlint is there so checker doesn't complain about it being a constant in an if.